.vscode/settings.json

```diff
@@ -0,0 +1,12 @@
+{
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "eslint.format.enable": true,
```

Comment:
Needed, otherwise eslint cannot be used as formatter. I never understood why this is not on by default.